content/docs/1.7.0/references/settings.md

```diff
@@ -472,6 +473,12 @@ Number of seconds that Longhorn allows for the completion of replica rebuilding
 
 Setting that allows you to enable the V1 Data Engine.
 
+#### RWX Volume Fast Failover
+
+> Default: `false`
+
+Enable improved ReadWriteMany volume HA by shortening the time it takes to recover from a node failure.
```

Comment:
NIT: I prefer to see a link here to the full feature description at `/high-availability/rwx-volume-fast-failover`, since https://github.com/longhorn/longhorn/wiki/Longhorn-Setting-Update#update-setting-page give us some leeway to expand on the setting descriptions here and I think it will be useful. I see that no other settings have links here though, so it's completely fine to ignore the suggestion.

Reply:
Yes, I thought about that and decided the same.  If some settings have links and discussion, they probably all merit it.